xontrib/cmd_done.py

```diff
@@ -90,6 +89,13 @@ def _linux_is_app_window_focused():
 
 def _darwin_is_app_window_focused():
     term = xsh.env.get("TERM_PROGRAM")
+    
+    if not term:
+        if ".pycharm" in xsh.env.get("__CFBundleIdentifier", ""):
+            term = "pycharm"
+        if ".Kate" in xsh.env.get("__CFBundleIdentifier", ""):
+            term = "kate"    
```

Comment:
I think this can be simplified into `xsh.env.get("__CFBundleIdentifier", "").split(".")[-1]` .